hack/generated/controllers/generic_controller.go

```diff
@@ -107,20 +107,48 @@ func (options *Options) setDefaults() {
 	}
 }
 
+func RegisterWebhooks(mgr ctrl.Manager, objs []runtime.Object) []error {
+	var errs []error
+
+	for _, obj := range objs {
+		if err := registerWebhook(mgr, obj); err != nil {
+			errs = append(errs, err)
+		}
+	}
+
+	return errs
+}
+
+func registerWebhook(mgr ctrl.Manager, obj runtime.Object) error {
+	_, err := conversion.EnforcePtr(obj)
+	if err != nil {
+		return errors.Wrap(err, "obj was expected to be ptr but was not")
+	}
+
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(obj).
+		Complete()
+}
+
 func RegisterAll(mgr ctrl.Manager, applier armclient.Applier, objs []runtime.Object, log logr.Logger, options Options) []error {
```

Comment:
Same comment about using `error` instead of `[]error` here.

Reply:
Yeah we can move to use `error` in both. Have done so.